### PR TITLE
[Rasheed]-Sage/automate flutter celo dapp deployment with GitHub actions

### PR DESCRIPTION
**Pull request by Taiwrash**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -25,7 +25,11 @@ import { sortBy } from "@site/src/utils/jsUtils";
  * - A short (≤120 characters) description of your project
  * - Use relevant tags to categorize your site (read the tag descriptions below)
  * - Add a local image preview (decent screenshot of your Docusaurus site)
- * - The image MUST be added to the GitHub repository, and use `require("img")`
+ * - The image MUST be added to the
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:28]:
```suggestion
 * - The image MUST be added to the  repository, and use `require("img")`
```

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -25,7 +25,11 @@ import { sortBy } from "@site/src/utils/jsUtils";
  * - A short (≤120 characters) description of your project
  * - Use relevant tags to categorize your site (read the tag descriptions below)
  * - Add a local image preview (decent screenshot of your Docusaurus site)
- * - The image MUST be added to the GitHub repository, and use `require("img")`
+ * - The image MUST be added to the 
+
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:29]:
```suggestion
```

File: src/data-tutorials/users.tsx [hunk 3]
```diff
@@ -25,7 +25,11 @@ import { sortBy } from "@site/src/utils/jsUtils";
  * - A short (≤120 characters) description of your project
  * - Use relevant tags to categorize your site (read the tag descriptions below)
  * - Add a local image preview (decent screenshot of your Docusaurus site)
- * - The image MUST be added to the GitHub repository, and use `require("img")`
+ * - The image MUST be added to the 
+ 
+
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:30]:
```suggestion
```

File: src/data-tutorials/users.tsx [hunk 4]
```diff
@@ -25,7 +25,11 @@ import { sortBy } from "@site/src/utils/jsUtils";
  * - A short (≤120 characters) description of your project
  * - Use relevant tags to categorize your site (read the tag descriptions below)
  * - Add a local image preview (decent screenshot of your Docusaurus site)
- * - The image MUST be added to the GitHub repository, and use `require("img")`
+ * - The image MUST be added to the 
+ 
+ 
+
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:31]:
```suggestion
```

File: src/data-tutorials/users.tsx [hunk 5]
```diff
@@ -25,7 +25,11 @@ import { sortBy } from "@site/src/utils/jsUtils";
  * - A short (≤120 characters) description of your project
  * - Use relevant tags to categorize your site (read the tag descriptions below)
  * - Add a local image preview (decent screenshot of your Docusaurus site)
- * - The image MUST be added to the GitHub repository, and use `require("img")`
+ * - The image MUST be added to the 
+ 
+ 
+ 
+ repository, and use `require("img")`
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:32]:
```suggestion
```

**Comment by developerkunal** [comment]
Fix the header image

File: blog/03-19-2023-automate-flutter-celo-dapp-deployment-with-github-actions/automate-flutter-celo-dapp-deployment-with-github-actions.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
+---
+title: Automate Flutter Celo DApp Deployment with GitHub Actions
+description: Learn how to deploy and publish your Flutter DApp to Google Play Store or Google Drive
+authors:
+  - name: Rasheed Mudasiru
+    title: Software/DevOps Engineer
+    url: https://github.com/Taiwrash
+    image_url: https://avatars.githubusercontent.com/u/49725691?v=4
+tags: [celo sage, dapp, intermediate, celo, flutter, github actions]
```
Comment (developerkunal) [inline blog/03-19-2023-automate-flutter-celo-dapp-deployment-with-github-actions/automate-flutter-celo-dapp-deployment-with-github-actions.md:9]:
```suggestion
tags: [celosage, dapp, intermediate, celo, flutter]
```

**Comment by Taiwrash** [comment]
> Fix the header image

Thanks @developerkunal  I haven't gotten it from the team @joenyzio

File: src/data-tutorials/users.tsx [hunk 7]
```diff
@@ -25,7 +25,7 @@ import { sortBy } from "@site/src/utils/jsUtils";
  * - A short (≤120 characters) description of your project
  * - Use relevant tags to categorize your site (read the tag descriptions below)
  * - Add a local image preview (decent screenshot of your Docusaurus site)
- * - The image MUST be added to the GitHub repository, and use `require("img")`
+ * - The image MUST be added to the  repository, and use `require("img")`
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:28]:
```suggestion
 * - The image MUST be added to the GitHub repository, and use `require("img")`
```

**Comment by developerkunal** [comment]
@Taiwrash don't forgot to remove old old image which is not going to be in use

**Comment by Taiwrash** [comment]
> @Taiwrash don't forgot to remove old old image which is not going to be in use

All  fixed and added

**Review by developerkunal — APPROVED** [review]
Approved

**Comment by Taiwrash** [comment]
Thanks @developerkunal Appreciated everything

**Review by Ishanpathak1 — APPROVED** [review]
approved @developerkunal

**Pull request merged**